src/docs/about.md

```diff
@@ -0,0 +1,208 @@
+ES5 that runs in the browser), the output of that is passed to the `uglify()` plugin which will minify our JS 
+into a smaller format. The output of `uglify()` will in turn be passed as 1 input to `merge()`.
+
+Additionally, a `src/styles` directory and the input file `site.scss` is passed to `sass()` which will convert your
+`.scss` files into `.css` files, its output (your css) is passed as an input to `autoprefixer()` which will add
+vendor  prefixes (like `-ms` or `-webkit`) to attributes, which is then in turn passed into the `merge()` plugin.
+
+The `merge()` plugin will copy the contents of each of its inputs into its output directory. Thus, it merges our 
+uglified Javascript and out vendor prefixed css. This then becomes our final output and is what is written to 
+our target (destination) directory.
+
+This should all be fairly familiar to you if you've ever written Javascript (or any programming language for that
+matter) before, it's just inputs and output.
+
+## Thinking in Broccoli
+
+There are 3 main concepts to get your head around when using Broccoli:
+
+* [Plugins](#plugins)
+* [Nodes](#nodes)
+* [Trees](#trees)
+
+### Plugins
+
+Plugins are what a build pipeline developer will interact with most. Plugins are what do the actual work of
+transforming input files into output files. The API of a plugin is pretty simple, all that's required is creating
+a class that extends the [broccoli-plugin](https://github.com/broccolijs/broccoli-plugin) base class, and
+implementing a `build()` method, that performs some work or returns a promise.
+
+```js
+const Plugin = require('broccoli-plugin');
+
+class MyPlugin extends Plugin
+{
+  build() {
+    // A plugin can receive single or multiple inputs nodes/directories
+    // The files are available in this.inputPaths[0]/this.inputPaths[1]...
+    // You can do anything with these files that you can do in node
+    // The output of your plugin must write to this.outputPath
+  }
+}
+```
+
+Basic plugins are super simple, anything you can do in node, you can do in the `build()` method. A plugin can
+receive one or multiple inputs, and these are available in the `this.inputPaths` array in the order they are 
+provided. `this.inputPaths` contains paths to directories, that are the `outputPath` of previous plugins. Each 
+`inputPath` contains files that you can manipulate and write to `this.outputPath`. Broccoli will handle the 
+state of these directories and passing them between plugins.
+
+There is a special case where a `string` is passed as an input to a plugin. When parsing your build pipeline, 
+Broccoli will automatically convert a string input into a
+[source plugin](https://github.com/broccolijs/broccoli-source). This plugin basically connects its input directory
+directly to its output directory, and optionally allows Broccoli to be notified when files within the input 
+directory change and trigger a rebuild.
+
+### Nodes
```

Comment:
Do you think we can get away without talking about nodes? I think for the most part, just thinking about broccoli-plugins being connected by broccoli in a graph is sufficient. Yes that implies broccoli-plugins are nodes, but I worry raising nodes and pointing to graph-theory provides less value than cost.

Reply:
Yeah I think that's fair.